uint8 x_pos
uint8 y_pos
uint8 zoom
